### ISSUE TITLE
new kpatch subpackage coming in rhel-8.4, 9.0

### ISSUE THREAD

**Issue by joe-lawrence** [minimization/content-resolver-input#268]
Hi, the kpatch team intends to introduce a new subpackage to the `kpatch` package called `kpatch-dnf` in RHEL-8.4.

We've already added [sst_kernel_ft-kpatch.yaml](https://github.com/minimization/content-resolver-input/blob/master/configs/sst_kernel_ft-kpatch.yaml) for the main package, will we need update this file if/when a new subpackage is added?

Thanks
-- Joe

cc: @julien-thierry

**Comment by joe-lawrence** [comment]
Adding update from Adam over email:
```
Hi Joe,

If you're adding it to RHEL 9, too, then yes, please, it needs to be listed in a Content Resolver workload.

Cheers,
Adam
```

**Comment by joe-lawrence** [comment]
@asamalik : We're building these .el9 packages now.  What's the proper way to add a RHEL-only 'kpatch-dnf' subpackage to our yaml file?  Would these need a new entry in the package_placeholders?   Thanks, -- Joe

**Comment by asamalik** [comment]
@joe-lawrence yes, please. Content Resolver only pulls in what's in the workload yaml configs.

**Comment by joe-lawrence** [comment]
I noticed an automated errata was created (I'm assuming as a result of merging the related PR): https://errata.devel.redhat.com/advisory/66900  however it states that the build is "Missing 1 product listing, The brew files are incomplete for the following build and need to be reloaded: kpatch-0.9.2-1.el9.1."   Is this expected and what do we need to do to satisfy the product listing requirement?  Thanks.

**Comment by jwboyer** [comment]
It'll get sorted out when we have an internal compose available.  You shouldn't have to do anything.

**Comment by joe-lawrence** [comment]
> It'll get sorted out when we have an internal compose available. You shouldn't have to do anything.

Works for me, thanks  :+1:

**Comment by joe-lawrence** [comment]
@jwboyer : apologies for bumbling through this process, feel free to redirect to email or RTFM if there is one available :)

Today I looked at the latest internal 9.0.0 development compose directory and still don't see the new subpackage.  And for errata 66900, I reloaded the files for the currently attached build, now says that the subpackage in question "RPM not in Product Listings".  (Similar to before, but the message looks slightly different this time.)   I'm not really sure what the next step is for adding this package to 9.0.0.

We will probably expect a new package build in the next week (rhbz-1912457), should we attach that one to the errata 66900 and then move to QE?  Thanks.
